.github/workflows/amiberry.yml

```diff
@@ -11,8 +11,13 @@ on:
 concurrency:
   group: ${{ github.workflow }}-${{ github.event.inputs.platform }}-${{ github.event.inputs.distro }}
   cancel-in-progress: true
+permissions:
+  contents: read
```

Comment:
Is there a reason for granting contents read permissions here? This workflow doesn't clone the repo, so `permissions: {}` should work instead to deny any token permissions. Same for `dietpi-build.yml`.

Reply:
Made it now like that in `dev` branch, i.e. zero token permissions for image and Amiberry build workflows, which works fine 🙂.